test/blackbox/common/DDSBlackboxTestsBasic.cpp

```diff
@@ -66,15 +67,16 @@ using ReturnCode_t = eprosima::fastrtps::types::ReturnCode_t;
  */
 TEST(DDSBasic, WarningOnDelete)
 {
+    using namespace eprosima::fastrtps;
     namespace dds = eprosima::fastdds::dds;
     auto factory = dds::DomainParticipantFactory::get_instance();
 
     // Set intraprocess delivery to full
-    LibrarySettings library_settings;
+    LibrarySettingsAttributes library_settings;
+    library_settings = xmlparser::XMLProfileManager::library_settings();
```

Comment:
here the order is inverted

Reply:
We are fixing here something we did wrong in master